feature_engine/timeseries/forecasting/lag_features.py

```diff
@@ -95,6 +95,27 @@ class LagFeatures(BaseForecastTransformer):
     See Also
     --------
     pandas.shift
+
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> from feature_engine.timeseries.forecasting import LagFeatures
+    >>> X = pd.DataFrame(dict(date = ["2022-09-18",
+    >>>                             "2022-10-27",
+    >>>                             "2022-12-24",
+    >>>                             "2023-01-09",
+    >>>                             "2023-02-12"],
+    >>>                     x1 = [1,2,3,4,5],
+    >>>                     x2 = [6,7,8,9,10]
+    >>>                     ))
+    >>> lf = LagFeatures(periods=[1,2])
+    >>> lf.fit_transform(X)
+            date  x1  x2  x1_lag_1  x2_lag_1  x1_lag_2  x2_lag_2
+    0  2022-09-18   1   4       NaN       NaN       NaN       NaN
+    1  2022-10-27   2   5       1.0       4.0       NaN       NaN
+    2  2022-12-24   3   6       2.0       5.0       1.0       4.0
+    3  2023-01-09   4   7       3.0       6.0       2.0       5.0
```

Comment:
Is the output correct? where is the 4 in x2_lag_2 coming from? there is no 4 in the original variable.

Also, could we make the timestamps consecutive? each one a different  but consecutive day? it's more representative of forecasting data like sales or stocks

Reply:
It is correct. I'm asking for lags 1 and 2. So the 4 is the first row, from column x2 shifted two periods. 
Fixing dates.